bin/build-ci

```diff
@@ -26,7 +26,8 @@ class Project
       new('backend', weight: 215),
       new('backend', test_type: :teaspoon, title: "backend JS", weight: 15),
       new('core', weight: 220),
-      new('sample', weight: 22)
+      new('sample', weight: 22),
+      new('legacy_promotions', weight: 22)
```

Comment:
@elia can you tell martin, what would be a good weight value here?

Reply:
@tvdeyen @mamhoff not that I'm too fond of these hardcoded weights, but I'd say that should relatively represent the time a particular suite takes. 

We might even refresh all the numbers and make them just the number of seconds of the last run on the main branch, e.g. [core took 9min](https://app.circleci.com/pipelines/github/solidusio/solidus/6090/workflows/ad3cd86b-abfd-4f8a-9585-d75b273f33cf/jobs/56170/parallel-runs/0/steps/0-118) so the weight should be `9 * 60 = 540`, that's an easy way to also come up with a sensible number for the new suite.